### PR TITLE
Certfile keyfile generated

### PR DESCRIPTION
**Pull request by keevBush**
- Display error when failed load keypair
- Generate keypair when cert.pem and key.pem files exist both

### CODE REVIEW

**Comment by keevBush** [comment]
Last work to display Folder name into a restoreversion modal is in this pull request

**Comment by imsodin** [comment]
Please keep the two separated, i.e. dialog naming in #6406 and the key errors/generation in this PR.

File: cmd/stdiscosrv/main.go [hunk 1]
```diff
@@ -99,11 +106,15 @@ func main() {
 
 	cert, err := tls.LoadX509KeyPair(certFile, keyFile)
 	if err != nil {
-		log.Println("Failed to load keypair. Generating one, this might take a while...")
-		cert, err = tlsutil.NewCertificate(certFile, keyFile, "stdiscosrv", 20*365)
-		if err != nil {
-			log.Fatalln("Failed to generate X509 key pair:", err)
+		log.Println("Failed to load keypair because:", err)
+		if !fileExists(certFile) && !fileExists(keyFile) {
```
Comment (calmh) [inline cmd/stdiscosrv/main.go]: I'd go with just `if os.IsNotExist(err)` here; that should be sufficient

Comment (keevBush) [reply cmd/stdiscosrv/main.go]: for cert.pem or for key.pem or for both files?

Comment (calmh) [reply cmd/stdiscosrv/main.go]: It's ok to just check the returned error from `LoadX509KeyPair`. If it's `os.IsNotExist` then we know that either the key or the cert didn't exist, and we can generate new ones.

Comment (keevBush) [reply cmd/stdiscosrv/main.go]: I fix that

File: gui/default/syncthing/folder/restoreVersionsModalView.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<modal id="restoreVersions" status="default" icon="fas fa-undo" heading="{{'Restore Versions' | translate}}" large="yes" closeable="yes">
```
Comment (calmh) [inline gui/default/syncthing/folder/restoreVersionsModalView.html]: gui changes should still not be in this PR

Comment (keevBush) [reply gui/default/syncthing/folder/restoreVersionsModalView.html]: It's on another branch. "dialog-titre"

**Pull request closed without merging**